packages/components/src/components/DaInsaneAd.vue

```diff
@@ -1,14 +1,15 @@
 <template>
-  <div class="insane insane--ad">
-    <a :href="ad.link" target="_blank" class="insane__link" @click="$emit('click', ad)">
+  <div class="insane insane--ad hide-menu" :class="cls">
```

Comment:
i had to add a `hide-menu` class, since it pushes to left and leaving extra white space for the menu since menu configuration was not mentioned.